docs/iris/src/whatsnew/contributions_2.3.0/newfeature_2019-Feb-25_compare_nan.txt

```diff
@@ -0,0 +1,2 @@
+* :func:`iris.util.array_equal` now has a 'withnans' keyword, which provides
+  a NaN-tolerant array comparison.
```

Comment:
You've targetted this at the `v2.2.x` branch but included a whats new entry to the `2.3.0` whats new?

Reply:
D'oh!

Reply:
Addressed .. ?